ng-gc-base-app-service/ng-gc-base-app-service.spec.js

```diff
@@ -21,21 +21,20 @@ describe('BaseAppService', function() {
     }));
   });
 
-  describe('RavenInitializer', function() {
+  describe('BugsnagInitializer', function() {
     beforeEach(function() {
-      $window.Raven = {
-        setUser: function() {}
+      $window.Bugsnag = {
+        user: null
       };
     });
-    beforeEach(module('ngGcRavenInitializer'));
+    beforeEach(module('ngGcBugsnagInitializer'));
     injectDependencies();
 
-    it('Raven.setUser', function() {
-      var user = {id: 1};
-      spyOn($window.Raven, 'setUser');
+    it('Bugsnag.user', function() {
```

Comment:
would rather have a more descriptive `it`. `it('sets the bugsnag user correctly')` ?